owlapi/src/main/java/org/aksw/owlapi/OwlapiApplication.java

```diff
@@ -0,0 +1,43 @@
+package org.aksw.owlapi;
+
+import org.aksw.owl2nl.OWLAxiomConverter;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.net.URLDecoder;
+import java.nio.charset.StandardCharsets;
+
+@SpringBootApplication
+@RestController
+public class OwlapiApplication {
+
+	public static void main(String[] args) {
+		SpringApplication.run(OwlapiApplication.class, args);
+	}
+
+	@GetMapping("/hello")
+	public String hello(@RequestParam(value = "name", defaultValue = "World") String name) {
+		return String.format("Hello from owlApi %s!", name);
+	}
+
+	@GetMapping("/getOntology")
+	public String GetOntology(@RequestParam(value = "path", defaultValue = "") String path) {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.